nifi-nar-bundles/nifi-shopify-bundle/nifi-shopify-processors/src/main/java/org/apache/nifi/processors/shopify/model/ResourceDirectory.java

```diff
@@ -0,0 +1,189 @@
+                    "Get products in a merchant's store ",
+                    IncrementalLoadingParameter.UPDATED_AT,
+                    PAGINATION_SUPPORTED
+            ), ShopifyResource.newInstance(
+                    "smart_collections",
+                    "Smart Collections",
+                    "A smart collection is a grouping of products defined by rules that are set by the merchant.",
+                    IncrementalLoadingParameter.UPDATED_AT,
+                    PAGINATION_SUPPORTED
+            )));
+    public static final List<ShopifyResource> SALES_CHANNEL_RESOURCES =
+            Collections.unmodifiableList(Arrays.asList(ShopifyResource.newInstance(
+                    "collection_listings",
+                    "Collection Listings",
+                    "A CollectionListing resource represents a product collection that a merchant has made available to your sales channel.",
+                    IncrementalLoadingParameter.NONE,
+                    PAGINATION_SUPPORTED
+            ), ShopifyResource.newInstance(
+                    "product_listings",
+                    "Product Listings",
+                    "A ProductListing resource represents a Product which is available to your sales channel.",
+                    IncrementalLoadingParameter.UPDATED_AT,
+                    PAGINATION_SUPPORTED
+            )));
+    public static final List<ShopifyResource> STORE_PROPERTY_RESOURCES =
+            Collections.unmodifiableList(Arrays.asList(ShopifyResource.newInstance(
+                    "countries",
+                    "Countries",
+                    "The Country resource represents the tax rates applied to orders from the different countries where a shop sells its products.",
+                    IncrementalLoadingParameter.NONE,
+                    PAGINATION_NOT_SUPPORTED
+            ), ShopifyResource.newInstance(
+                    "currencies",
+                    "Currencies",
+                    "Merchants who use Shopify Payments can allow customers to pay in their local currency on the online store.",
+                    IncrementalLoadingParameter.NONE,
+                    PAGINATION_NOT_SUPPORTED
+            ), ShopifyResource.newInstance(
+                    "policies",
+                    "Policies",
+                    "Policy resource can be used to access the policies that a merchant has configured for their shop, such as their refund and privacy policies.",
+                    IncrementalLoadingParameter.NONE,
+                    PAGINATION_NOT_SUPPORTED
+            ), ShopifyResource.newInstance(
+                    "shipping_zones",
+                    "Shipping Zones",
+                    "ShippingZone resource can be used to view shipping zones and their countries, provinces, and shipping rates.",
+                    IncrementalLoadingParameter.UPDATED_AT,
+                    PAGINATION_NOT_SUPPORTED
+            ), ShopifyResource.newInstance(

```

Comment:
Interesting, directly calling the .../admin/api/2022-10/shipping_zones.json endpoint returns a json.

Reply:
It does not return a zip for me but the problem is that the Json parser expects that the endpoints contain an array of objects. The 'Shop' endpoint is special because there is only on object wrapped under the 'shop' field.

"shop": {
        "id": 65529675995,
        "name": "nifi-test",
        "email": "lehelb@cloudera.com",
        ...
}
I think it's better to remove it for now.